### PR TITLE
Expand fs support with access & faccessat hooks for absolute paths

### PR DESCRIPTION
**Pull request by DmitryDodzin**
Support hooks for `access` & `faccessat` only for absolute paths

Relevant for #200 and #137

### CODE REVIEW

**Review by aviramha — APPROVED** [review]
LGTM! Please wait for review from Eyal before merging.

File: mirrord-layer/src/file.rs
Comment (eyalb181) [inline mirrord-layer/src/file.rs:372]: Typo here in the arg name

**Review by eyalb181 — APPROVED** [review]
Approved, just fix the typo. Thanks!

**Pull request merged**
